src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Exception {
+    public AuthenticationException(String message) {
+        super(message);
+    }
```

Comment:
I recommend you throw exceptions in each case. So better don't use this constructor

Reply:
![image](https://user-images.githubusercontent.com/110828777/206521042-25b66a0c-f365-46c4-ab9b-ad45fc860910.png)
how we shoult throw exception here?